VictoriaAlbert/VictoriaAlbert/Extensions.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Extensions.swift
+//  PokeDex
+//
+//  Created by Jason Wang on 10/15/16.
+//  Copyright © 2016 Jason Wang. All rights reserved.
+//
+
+import UIKit
+
+let cacheImage = NSCache<NSString, UIImage>()
+extension UIImageView {
```

Comment:
I would us not to use this extension any longer, depending on our URLSession based data loading method in APIManager.